src/noit_filters.c

```diff
@@ -186,7 +189,12 @@ noit_filter_compile_add(mtev_conf_section_t setinfo) {
       rule->type = (!strcmp(buffer, "accept") || !strcmp(buffer, "allow")) ?
                      NOIT_FILTER_ACCEPT : NOIT_FILTER_DENY;
     }
-
+    int32_t ffp = DEFAULT_FILTER_FLUSH_PERIOD;
+    if(!mtev_conf_get_int32(rules[j], "ancestor-or-self::node()/@filter_flush_period", &ffp))
```

Comment:
might want to name this `filter_flush_period_ms` or use the time based config here (`300s`)